kubetest/dind/util.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dind
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"k8s.io/test-infra/kubetest/util"
+)
+
+// GetDockerVersion reads the stable-status.txt file from bazel to get the docker artifact version.
+func GetDockerVersion() (string, error) {
+	fileName := util.K8s("kubernetes", "bazel-out", "stable-status.txt")
```

Comment:
looks like `GetDockerVersion()` can happen before `--extract`, what if user hasn't checkout k8s yet?

Reply:
I'll just ensure the image exists once Up is called.